lib/CleantalkSP/SpbctWP/Firewall/UploadChecker.php

```diff
@@ -301,6 +306,26 @@ public static function runCheckForWordpressModules(
         // Output result message
         if ( $overall_result ) {
             if (self::checkVulnerability($source)) {
+                $name_file = $_FILES['pluginzip']['name'];
+
+                ?>
+                <b>If you want to continue the installation, add the file and click install.</b>
```

Comment:
The text string have to convert to translate ready.

Reply:
It should be deleted altogether